components/automate-ui/src/app/modules/infra-proxy/org-details/org-details.component.ts

```diff
@@ -37,90 +23,45 @@ export type OrgTabName = 'cookbooks' | 'roles' | 'details';
 
 export class OrgDetailsComponent implements OnInit, OnDestroy {
   public org: Org;
-  public cookbooks: Cookbook[] = [];
-  public roles: InfraRole[] = [];
   public loading$: Observable<boolean>;
-  public sortedCookbooks$: Observable<Cookbook[]>;
-  public saveSuccessful = false;
-  public saveInProgress = false;
-  public isLoading = true;
-  public url: string;
   public serverId;
-  public OrgId;
-  public updateOrgForm: FormGroup;
-  public tabValue: OrgTabName = 'cookbooks';
+  public orgId;
```

Comment:
One of these isn't yours, but would be cool if you could add types for these while you're in here

Reply:
I have added types.